worker-configs/src/main/java/com/hpe/caf/configs/HealthConfiguration.java

```diff
@@ -0,0 +1,184 @@
+/*
+ * Copyright 2015-2024 Open Text.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hpe.caf.configs;
+
+import jakarta.validation.constraints.Min;
+
+/**
+ * Configuration for the liveness and readiness checks.
+ */
+public class HealthConfiguration
+{
+    /**
+     * The initial delay to use when first scheduling the liveness check.
+     */
+    @Min(0)
+    private int livenessInitialDelaySeconds = 15;
+
+    /**
+     * The interval on which to perform a liveness check for while in a healthy state.
+     */
+    @Min(60)
```

Comment:
Doesn't feel right for the minimum to be 60 here.  Same for a number of other configs also.

Reply:
I didn't want to allow users to be able to set this to a very low value, as that would result in the checks getting run continuously, which I though could be a performance problem. I choose 60 as I think that is the lowest we currently use for `healthChecks.intervalSeconds` in Verity/deploy. What would be a better minimum value here, 10 seconds?

Reply:
I'd take it all the way down to 1 or 2.  It is just a minimum &mdash; it's not like you're advising that configuration.